### PR TITLE
Add note on `CommandResults` `readable_output` arg

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Status
Ready

## Related Issues
N/A

## Description
Small change: added a note in the row of the `CommandResults` table for the `readable_output` arg to specify how to create a War Room entry only, and not write any data to context, because I had a client ask about this. Feel free to rephrase or move the note to a different part of the page, as you see fit.

## Screenshots
N/A

### CODE REVIEW

File: docs/integrations/code-conventions.md [hunk 1]
```diff
@@ -744,7 +744,7 @@ This class is used to return outputs. This object represents an entry in warroom
...
 | outputs_key_field | str    | Primary key field in the main object. If the command returns Incidents, and of the properties of Incident is incident_id, then outputs_key_field='incident_id'                             |
 | outputs           | object | The data to be returned and will be set to context                                                                                                                                         |
-| readable_output    | str   | (Optional) markdown string that will be presented in the warroom, should be human readable -  (HumanReadable) - if not set, readable output will be generated via tableToMarkdown function |
+| readable_output    | str   | (Optional) markdown string that will be presented in the War Room, should be human readable -  (HumanReadable) - if not set, readable output will be generated via tableToMarkdown function. To create a War Room entry only, and not write any data to context, use `readable_output` and do _not_ use `outputs`. |
```
Comment (reutshal) [inline docs/integrations/code-conventions.md:747]: I don't think we should add this here, this is not related to the "readable_output" argument - 
If one doesn't want to write anything to the context they should just not pass the outputs argument to the commandResults struct.
Did you encounter any problem with the current implementation?

Comment (ali-sawyer) [reply docs/integrations/code-conventions.md:747]: I see what you mean. I opened this because I had a client ask about this use case – "is it possible to post a note to the war room without adding content to the context"? – and I thought the documentation could be clearer in describing how to implement this, since the documentation makes it look like the `outputs` arg is required.

Maybe we could add a simple code sample for this particular use case. Might be better on https://xsoar.pan.dev/docs/integrations/context-and-outputs. What do you think?

Comment (reutshal) [reply docs/integrations/code-conventions.md:747]: This is simply a bug in the documentation - the "outputs' field is indeed optional. I think this will make things clearer to the customer, WDYT ?

Comment (ali-sawyer) [reply docs/integrations/code-conventions.md]: Yes agreed, this looks good to me.

**Comment by anara123** [comment]
what is the status here?

**Review by reutshal — APPROVED** [review]
